src/ray/core_worker/test/core_worker_test.cc

```diff
@@ -30,6 +31,10 @@ std::string mock_worker_executable;
 
 ray::ObjectID RandomObjectID() { return ObjectID::FromRandom(); }
 
+inline TaskArg PassByValueWithBuffer(const std::shared_ptr<Buffer> &buffer) {
```

Comment:
Why "WithBuffer" ? Isn't it always with a buffer? Also, this function just wraps a single function call, better to call it directly.

Reply:
Changed to call it directly.